Name: Remove Clear Bookmarks Button
Version: 3.0.0-1
Author: Jason Robitaille
Description: Removes the Clear Bookmarks button from the preferences screen of the browser app  (MIT license)

--- .orig/usr/palm/applications/com.palm.app.browser/source/Preferences.js
+++ /usr/palm/applications/com.palm.app.browser/source/Preferences.js
@@ -39,7 +39,6 @@
 						{kind: "ToggleButton", name: "click2play", onChange: "togglePreferenceClick", preference: "click2play", type: "System", inverted: true}
 					]}
 				]},
-				{kind: "Button", caption: $L("Clear Bookmarks"), onclick: "promptButtonClick", dialog: "clearBookmarksPrompt"},
 				{kind: "Button", caption: $L("Clear History"), onclick: "promptButtonClick", dialog: "clearHistoryPrompt"},
 				{kind: "Button", caption: $L("Clear Cookies"), onclick: "promptButtonClick", dialog: "clearCookiesPrompt"},
 				{kind: "Button", caption: $L("Clear Cache"), onclick: "promptButtonClick", dialog: "clearCachePrompt"},
